templates/field/field--oe-contact--oe-press-contact-url--default.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * @file
+ * Press contacts field in the Contact entity default view mode.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<div class="ecl-u-border-top ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-mt-s ecl-u-pt-l ecl-u-pb-l">
```

Comment:
I've implemented as described on the https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=437851760 page "Border (.ecl-u-border-top .ecl-u-border-bottom ecl-u-border-color-grey-15)".

<img width="575" alt="Screenshot 2020-07-31 at 16 23 11" src="https://user-images.githubusercontent.com/1183951/89040366-57aaad80-d34c-11ea-8bdd-a1430f1453a9.png">

But due to "Press contacts" and "Send a message" fields are optional there can be different variations of top and bottom border.